desktop/src/main/nl/tudelft/ti2806/riverrush/graphics/entity/Animal.java

```diff
@@ -46,6 +46,7 @@ public AnimalActor getActor() {
      */
     public void setActor(final AnimalActor aActor) {
         this.actor = aActor;
+        this.actor.setAnimal(this);
```

Comment:
Not a fan of this. It seems animal is only used in actor for collision, can it not be moved to animal then instead?